PyPDF2/_writer.py

```diff
@@ -411,6 +411,54 @@ def insertBlankPage(
         deprecate_with_replacement("insertBlankPage", "insert_blank_page")
         return self.insert_blank_page(width, height, index)
 
+    @property
+    def opening(self) -> Union[None, Destination, TextStringObject, ByteStringObject]:
+        """
+        property returning the opening Destination
+        returns None if no destination is set
+
+        (value stored in "/OpenAction" entry in the Pdf Catalog)
```

Comment:
Maybe something like:
```suggestion
        Property returning the opening destination ("/OpenAction" entry in the
        PDF catalog). Set to `None` to default to opening at top of first page.
```